Add Support for Passing in Metadata as JSON String (#25)
pytest-dev/pytest-metadata: README.rst

   pytest --metadata foo bar --metadata baz zoo

> If you have lots of metadata, you can also provide a json string::

Reviewer: ```suggestion
There's also the possibility of passing in metadata as a JSON string::
```

A more neutral wording.
